.github/workflows/CI.yml

```diff
@@ -125,11 +125,10 @@ jobs:
 
   coverage:
     runs-on: ubuntu-latest
-    if: ${{ !github.event.pull_request.draft }}    
+    if: ${{ !github.event.pull_request.draft }}
     env:
       CARGO_TERM_COLOR: always
     steps:
-      - uses: actions/checkout@v4
```

Comment:
You'll still need this 😅

Reply:
whoops, that was a bad rebase I guess. You committed the changes and I committed the changes at the same time, so I rebased